idaes/generic_models/properties/core/generic/generic_property.py

```diff
@@ -832,14 +836,16 @@ def define_metadata(cls, obj):
         # TODO : Need to fix to have methods for things that may or may not be
         # created by state var methods
         obj.add_properties(
-            {'flow_mol': {'method': None},
+            {'flow_mol': {'method': 'flow_mol'},
```

Comment:
Not totally sure, but doesn't the flow\_mol method here create a component with the same name, so would replace itself?